### PR TITLE
feat: add games201 release test

### PR DESCRIPTION
**Pull request by L2ncE**
refer to https://github.com/taichi-dev/taichi/issues/5741

### CODE REVIEW

File: timelines/games201/mass_spring_explicit.yml [hunk 1]
```diff
@@ -0,0 +1,524 @@
+---
+- path: /Users/lance/python/taichi-release-tests/repos/games201/mass_spring_explicit.py
```
Comment (feisuzhu) [inline timelines/games201/mass_spring_explicit.yml]: Should use relative path

File: timelines/games201/mass_spring_explicit.yml [hunk 2]
```diff
@@ -0,0 +1,524 @@
...
+  args: []
+  steps:
+  - {frame: 1, action: move, position: [0.561, 0.812]}
+  - {frame: 2, action: move, position: [0.561, 0.846]}
+  - {frame: 1, action: move, position: [0.564, 0.877]}
+  - {frame: 1, action: move, position: [0.572, 0.893]}
+  - {frame: 1, action: move, position: [0.58, 0.908]}
+  - {frame: 1, action: move, position: [0.592, 0.92]}
+  - {frame: 1, action: move, position: [0.598, 0.926]}
```
Comment (feisuzhu) [inline timelines/games201/mass_spring_explicit.yml]: Please cut redundant moves and actions as much as possible, to reduce running time.

**Review by feisuzhu — CHANGES_REQUESTED** [review]

File: timelines/games201/mass_spring_explicit.yml (hunk 1 above)
Comment (L2ncE) [reply timelines/games201/mass_spring_explicit.yml]: sorry forgot to change

File: timelines/games201/mass_spring_explicit.yml (hunk 2 above)
Comment (L2ncE) [reply timelines/games201/mass_spring_explicit.yml]: got it, thx

Comment (L2ncE) [reply timelines/games201/mass_spring_explicit.yml]: done

**Comment by L2ncE** [comment]
@feisuzhu PTAL

**Review by feisuzhu — APPROVED** [review]
LGTM!

**Pull request merged**